app/controllers/application_controller.rb

```diff
@@ -23,8 +23,23 @@ def logged_user?
     flash[:warning] = t "please_log_in"
   end
 
+  def admin?
+    if logined?
+      return if current_user.admin? || current_user.staff?
+
+    end
+
+    flash[:warning] = t "not_permission"
+    redirect_to root_path
+  end
+
   def load_date_params
     params[:start_date] ||= current_date
     params[:end_date] ||= current_date
   end
+
+  def update_bookings
+    bookings = Booking.pending.update_status_list
```

Comment:
tên scope gì lạ thế, update_status_list mà thấy có update đâu.

Reply:
cái ni update status khi booking quá hạn chưa được thanh toán as

Reply:
a có thấy em gọi hàm update hay save trong scope đâu